lib/IRGen/GenCall.cpp

```diff
@@ -5500,8 +5501,9 @@ Address irgen::emitAutoDiffProjectTopLevelSubcontext(
   return Address(call, IGF.IGM.Int8Ty, IGF.IGM.getPointerAlignment());
 }
 
-Address irgen::emitAutoDiffAllocateSubcontext(
-    IRGenFunction &IGF, Address context, llvm::Value *size) {
+Address irgen::emitAutoDiffAllocateSubcontext(IRGenFunction &IGF,
```

Comment:
Do we need to keep these functions after all? We are not going to call these builtins from the compiler.

Reply:
Removed